src/map/enmity_container.cpp

```diff
@@ -137,6 +137,17 @@ float CEnmityContainer::CalculateEnmityBonus(CBattleEntity* PEntity)
         {
             enmityBonus -= PChar->PMeritPoints->GetMeritValue(MERIT_MUTED_SOUL, PChar);
         }
+        // Enmity Bonus for Tank Stance type effects  (Use of || Prevents multiple effects stacking with one another.)
+        if (PChar->StatusEffectContainer->HasStatusEffect(EFFECT_DEFENDER) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_FAN_DANCE) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_MAJESTY) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_SEIGAN) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_COUNTERSTANCE) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_DEFENSE_BOOST) ||
+            PChar->StatusEffectContainer->HasStatusEffect(EFFECT_YONIN))
+        {
+            enmityBonus += 0.25;
```

Comment:
Sorry, reviewed this a bit late. enmityBonus is an integer. Will need to have another var as a float that we can use as a multiplier in "bonus ="

Reply:
That's my fault.  I should have confirmed the variable type.  I'll work up a revised pr.